.agents/checks/eslint.strict.config.js

```
@@ -0,0 +1,60 @@
+/**
+ * Strict ESLint config for the sonarjs-lint review check.
+ *
+ * Uses eslint-plugin-sonarjs to get SonarQube-grade analysis without a server.
+ * This config is NOT used for regular development linting — only for the
+ * code review checks' static analysis pass.
+ *
+ * Install: pnpm dlx eslint-plugin-sonarjs
+ * Run:     pnpm dlx eslint --no-eslintrc --config .agents/checks/eslint.strict.config.js --ext .ts,.js,.vue {files}
+ */
+
+import sonarjs from 'eslint-plugin-sonarjs'
```

ESLint

[error] Unable to resolve path to module 'eslint-plugin-sonarjs'.

(import-x/no-unresolved)